examples/bri-3/src/bri/zeroKnowledgeProof/api/anchorHash.controller.ts

```diff
@@ -1,24 +1,12 @@
 import { Body, Controller, Post } from '@nestjs/common';
 import { CommandBus } from '@nestjs/cqrs';
-import { CreateAnchorHashCommand } from '../capabilities/createAnchorHash/createAnchorHash.command';
 import { VerifyAnchorHashCommand } from '../capabilities/verifyAnchorHash/verifyAnchorHash.command';
-import { CreateAnchorHashDto } from './dtos/request/createAnchorHash.dto';
```

Comment:
this dto can be removed too?

Reply:
CreateAnchorHashDto is only used in createAnchorHash.dto.ts and its test. I have removed these two files and committed the change to this PR.